### PR TITLE
Standardize docs formatting

### PR DESCRIPTION
**Pull request by zsd4yr**
Inconsistency in formatting of markdown files in the repo. I used [markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) to normalize formatting in all source files and [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) to fix all current spelling errors, clarify wording, insert oxford commas, remove contractions; also move all links to bottom

### CODE REVIEW

**Comment by zsd4yr** [comment]
@karelz If you don't like the link stuff, it's part of a single commit so we can discuss not doing it --however, I think it is a LOT more organized, especially if we use a link more than once in the same doc

**Comment by AdamYoblick** [comment]
:shipit:

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
 
-See [Contributing Guide](../Documentation/contributing.md) for information about:
+See [Contributing Guide][contributing] for information about:
 
 * Which kind of PRs we accept/reject for .NET Core 3.0 release
 * Coding style
 * PR style preferences (squashing vs. merging, etc.)
 * Developer guide for building and testing locally
+
+[comment]: <> (Links)
```
Comment (sharwell) [inline .github/CONTRIBUTING.md]: 😕 What does this do?

File: Documentation/contributing.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 # Contributing Guide
 
-The primary focus of .NET Core 3.0 release for Windows Forms is to achieve parity with .NET Framework. Priority will be given to changes that align with that goal. See the [roadmap](../roadmap.md) to understand project goals.
+The primary focus of .NET Core 3.0 release for Windows Forms is to achieve parity with .NET Framework. Priority will be given to changes that align with that goal. See the [roadmap][roadmap] to understand project goals.
```
Comment (sharwell) [inline Documentation/contributing.md]: 💡 I believe if the anchor is the same as the text, you can use the form `[roadmap][]`

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (zsd4yr) [reply .github/CONTRIBUTING.md]: It's a comment; there might be a better way to do that

File: Documentation/contributing.md (hunk 2 above)
Comment (zsd4yr) [reply Documentation/contributing.md]: That's good to know. For now I'd like to be explicit

**Comment by zsd4yr** [comment]
@OliaG please take some time to review today. 😄

**Comment by zsd4yr** [comment]
@karelz @OliaG I have made the change to put the single-use internal links (read: the short links) back in-line. As we discussed, I am not a fan of this change, and I think --as we discussed-- you will find it to be a confusing developer experience.

The rules as we agreed yesterday are now:
1. put long (read: external) links at bottom
2. put internal links in-line
  - unless they are used more than once; then put them at the bottom

I would like your blessings to return to when the links were all at the bottom

**Comment by zsd4yr** [comment]
@karelz is this good to go?

**Comment by AdamYoblick** [comment]
Karel, I'm merging this because I believe Zach made the requested changes and you're OOF. If there are additional changes you'd like to see, please let us know. :)

**Pull request merged**